types/filters/parse.go

```diff
@@ -221,6 +221,24 @@ func (filters Args) ExactMatch(field, source string) bool {
 	return false
 }
 
+// OnlyOneExactMatch returns true if there is only one filters and the source matches exactly this one.
+func (filters Args) OnlyOneExactMatch(field, source string) bool {
```

Comment:
Yeah I'm bad with naming :scream:. I feel like `MatchSingleValue` would be more appropriate indeed (but `UniqueExactMatch` would work well too :sweat_smile:).